src/homePage/homePage.component.tsx

```diff
@@ -0,0 +1,332 @@
+                    'Navigate through systems and their sub systems to view their component items.'
+                  }
+                </PaperDescription>
+                <Box marginTop="auto">
+                  <Button
+                    color="primary"
+                    variant="contained"
+                    component={Link}
+                    to={'/systems'}
+                    data-testid="systems-button"
+                  >
+                    {'Explore'}
+                  </Button>
+                </Box>
+              </Box>
+            </Paper>
+          </Grid>
+          <Grid item sm={12} md={4}>
+            <Paper sx={paperStyles} elevation={1}>
+              <Box sx={paperContentStyles}>
+                <PaperHeading variant="h4">{'Manufacturers'}</PaperHeading>
+                <PaperDescription variant="body1">
+                  {'View the manufacturers of parts used by EPAC.'}
+                </PaperDescription>
+                <Box marginTop="auto">
+                  <Button
+                    color="primary"
+                    variant="contained"
+                    component={Link}
+                    to={'/manufacturer'}
+                    data-testid="manufacturer-button"
+                  >
+                    {'Explore'}
+                  </Button>
+                </Box>
+              </Box>
+            </Paper>
+          </Grid>
+          <Grid item sm={12} md={4}>
+            <Paper
+              sx={{ ...paperStyles, backgroundColor: '#003088' }}
+              elevation={1}
+            >
+              <div
+                style={{
+                  backgroundImage: `url(${props.greenSwirl2Image})`,
+                  backgroundRepeat: 'no-repeat',
+                  backgroundPosition: 'top right',
+                  backgroundSize: 'auto 100%',
+                  height: '100%',
+                }}
+              >
+                <Box sx={paperContentStyles}>
+                  <BluePaperHeading variant="h4">{'EPAC'}</BluePaperHeading>
+                  <BluePaperDescription color="#ffffff" variant="body1">
+                    {
+                      'World-leading centre for scientific and industrial research.'
+                    }
+                  </BluePaperDescription>
+                  <Box marginTop="auto">
+                    <LightBlueButton
+                      color="primary"
+                      variant="contained"
+                      href={
+                        'https://www.clf.stfc.ac.uk/Pages/EPAC-Applications.aspx'
+                      }
+                      data-testid="facility-button"
+                    >
+                      {'Read more'}
+                    </LightBlueButton>
+                  </Box>
+                </Box>
+              </div>
+            </Paper>
+          </Grid>
+        </Grid>
+      </Box>
+    </div>
+  );
+};
+
+export interface HomePageProps {
+  pluginHost: String;
+}
```

Comment:
Can you obtain `pluginHost` from the settings file inside HomePage instead? It needs to be configurable for the production instance. In datagateway we were using the old style of redux (https://github.com/ral-facilities/datagateway/blob/develop/packages/datagateway-dataview/src/page/translatedHomePage.component.tsx#L45-L50), but I don't think redux should be necessary here.

Reply:
I'm struggling to find a way to get it, and get it to work as well

Reply:
Ah I see, yes its async. In that case I get the feeling we need to use redux in this case as well. I am having a chat with Joshua about this after lunch.

Reply:
We can avoid it for now using something like this
```
    // TODO: Replace this with redux if it ends up being used elsewhere
    const [pluginHost, setPluginHost] = useState<string>('');
    useEffect(() => {
      const loadPluginHost = async () =>
        setPluginHost((await settings)?.pluginHost || '');
      loadPluginHost();
    }, []);
```
and then we don't need the prop anymore either.